giraffe/src/utils/normalizeConfig.ts

```diff
@@ -16,7 +17,45 @@ export const normalizeLayers = (layers: LayerConfig[]): LayerConfig[] =>
     return layerConfig
   }) || []
 
-export const normalizeConfig = (config: Config): Config => ({
-  ...config,
-  layers: normalizeLayers(config?.layers),
-})
+export const normalizeConfig = (config: Config): Config => {
+  const dataProperties: Partial<Config> = {}
+
+  if (config) {
+    const {fluxResponse, fromFluxResult, table} = config
+
+    switch (config.layers?.[0]?.type) {
+      case 'simple table':
```

Comment:
Similar to how we put brackets around all `if` statements, even if it's not strictly necessary to guard against bugs, we should start getting into the habit of putting brackets around all `case` statements.

```ts
case 'simple table': {
  if (fromFluxResult) {
    ...
  }
  break
}